azkaban-web-server/src/web/js/azkaban/view/exflow.js

```diff
@@ -447,6 +449,10 @@ var updaterFunction = function() {
 			// 2 min updates
 			setTimeout(function() {updaterFunction();}, 2*60*1000);
 		}
+		else if (data.status == "KILLING") {
+			// 20 s updates
+			setTimeout(function() {updaterFunction();}, 30*1000);
```

Comment:
I'll fix the comment.

I set it to 30 s because if the flow is KILLING we can expect that it will finish ~soon. So have a shorter status polling interval.